api_tests/types/bn-chai/index.d.ts

```diff
@@ -1,11 +1,11 @@
-declare module "bn-chai" {
-    function bnChaiFn(ctor: any): Chai.ChaiPlugin;
-
-    export = bnChaiFn;
-}
-
-declare namespace Chai {
-    interface NumberComparer {
-        BN(value: any): void;
-    }
-}
+// declare module "bn-chai" {
```

Comment:
:mega: If we no longer need it, remove it please.

Reply:
Although I'd prefer to remove the `ts-ignore` and provide correct typings.

Reply:
Yes need to be removed.
I am trying to make this work before wasted my time on typings (and I did try to add typings for 10min then gave up).